### ISSUE TITLE
Auto layout issues

### ISSUE THREAD

**Issue by kevinrenskers** [soffes/SAMLabel#1]
If I use your label in a storyboard / nib file that uses Auto Layout and I then set the textEdgeInsets so the label gets some top and bottom padding, then I would expect this padding to show. Currently that's not the case.

This seems to fix that:

```
- (CGSize)intrinsicContentSize {
    CGSize size = [super intrinsicContentSize];
    return CGSizeMake(size.width, size.height+self.edgeInsets.top+self.edgeInsets.bottom);
}
```

**Comment by jpm** [comment]
This is a good change, except that you also need to adjust the width since there is left/right edge insets too.

``` objc
- (CGSize)intrinsicContentSize {
    CGSize size = [super intrinsicContentSize];
    return CGSizeMake(size.width + self.textEdgeInsets.left + self.textEdgeInsets.right, size.height + self.textEdgeInsets.top + self.textEdgeInsets.bottom);
}
```

**Comment by soffes** [comment]
@jpm can you PR that?

**Comment by jpm** [comment]
@soffes Sure, I just did a PR for that.

**Comment by soffes** [comment]
Fixed by #2
